scripts/functions.sh

```diff
@@ -70,3 +70,10 @@ function zipFilesWithPassword() {
        echo "$(datef) $CLIENT_PATH/client.zip with password protection has been generated"
     fi
 }
+
+function removeConfig() {
+    local CLIENT_ID="$1"
+
+    easyrsa revoke $CLIENT_ID
+    easyrsa gen-clr
```

Comment:
`gen-clr`  may be a typo, there is a `gen-crl` command that generates a certification revocation list